src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -1510,7 +1511,7 @@ public function getETags(): array
 
     public function isNoCache(): bool
     {
-        return $this->headers->hasCacheControlDirective('no-cache') || 'no-cache' == $this->headers->get('Pragma');
+        return $this->headers->hasCacheControlDirective('no-store') || 'no-store' == $this->headers->get('Pragma');
```

Comment:
This method is a getter that checks if the `no-cache` instruction is sent. If `no-store` includes `no-cache` in its spec, we can add a condition. The current condition must be kept.